internal/kafka/internal/presenters/data_plane_kafka_status.go

```diff
@@ -34,6 +34,7 @@ func ConvertDataPlaneKafkaStatus(status map[string]private.DataPlaneKafkaStatus)
 			KafkaVersion:    v.Versions.Kafka,
 			StrimziVersion:  v.Versions.Strimzi,
 			KafkaIBPVersion: v.Versions.KafkaIbp,
+			AdminServerURI:  v.Endpoint.AdminServerURI,
```

Comment:
this might change to 
```suggestion
			AdminServerURI:  v.AdminServerURI,
```
based on my question above on why we have the `endpoint` object